src/OMSimulatorLib/SystemTLM.h

```diff
@@ -57,6 +58,10 @@ namespace oms3
     SystemTLM& operator=(SystemTLM const& copy); ///< not implemented
 
   private:
+    void* model;
```

Comment:
Why do you use `void*`?

Reply:
The OMTLMSimulator API uses opaque pointers to keep track of the models. It could perhaps be done in a different way, but this has worked well so far.